flaggers/unobservedStop.py

```diff
@@ -16,15 +16,16 @@ def flag(self, data):
 
 		Args:
 			data (Object): data row from full dataset fetched from the db
+			config (Object): contains config vars
 
 		Returns: 
 			list: either empty or containing UNOBSERVED_STOP Flag
 
 		"""
 
 		#maxDistance specifies max distance away from the stop at which we start marking stop as an unobserved
-		#Instead of being hardcoded, in the future substitute to .env variable
-		maxDistance = 50
+		max_distance = config.get_value("unobserved_stop_distance")
+		if(max_distance == None) max_distance = 50
```

Comment:
Pytest compains about invalid syntax on line 28.

Reply:
ah, let me fix it, its what happens when switching back from node